src/containers/App.js

```diff
@@ -70,14 +72,26 @@ export default class App extends React.Component {
 
     const query = get('query', location) || {}
 
-    const version = Number(iOSAppVersion())
-    if (version < 1.7) {
+    const iOSVersion = iOSAppVersion()
+
+    const androidVersion = androidAppVersion()
+
+    if (iOSVersion > 0) {
+      trackEvent(OPENED_MOBILE_APP, {iOSVersion})
```

Comment:
Let's make these keys human-readable: "App Version (iOS)" and "App Version (Android)"